semver.py

```diff
@@ -164,6 +164,74 @@ def __iter__(self):
         for v in self._astuple():
             yield v
 
+    def bump_major(self):
+        """Raise the major part of the version
```

Comment:
I would emphasize in docstring that new object is returned instead of modifying `self`

Reply:
Good catch! :+1: What about this phrase:

```
Raise the major part of the version, return a new object
but leave self untouched
```

Reply:
Yes, it's very clear now, thanks :)